### PR TITLE
E2E tests for translated Cart & Checkout blocks

### LINKED ISSUE #5752: Critical flows: Shopper → Cart → Can view translated cart block
### Story

As a shopper, I want to see a translated cart block so that I understand all fields/options.

### File

`tests/e2e/specs/shopper/cart-translations.test.js`

### Precondition

Switch to a language that contains translations for the cart block, e.g. German. After switching to that language, fetch the translations for the WooCommerce Blocks plugin.

### Test

1. Go to the product page.
2. Add a product to the cart.
3. Go to the cart page.
4. Verify that all translatable fields show the corresponding translation.

### PR DESCRIPTION
**Pull request by alexflorisca**
This e2e test checks weather the Cart & Checkout are translated properly. Some notable thoughts on this:

It doesn't check every translatable string as this would take too much config and setup for different scenarios. For example, to test that "Create an account" link is translated, there's a lot of setup of changing WooCommerce settings, block settings, logging in and out, etc. This is a different test in itself. I aim to check the basics. If I've missed something crucial or you feel strongly we should be testing every scenario, let me know.

I've opted for fetching the language files from the test directly rather than in `wp-env-config.sh` as we can clean these up at the end of the test and reset the env as it was for the remaining tests.

Fixes #5752 and #5769

#### Automated Tests
* [x] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [x] E2E tests
### Manual Testing

How to test the changes in this Pull Request:

1. If E2E github actions pass in this PR then it's all good. 
2. You can alternatively run this specific test locally with `npm run test:e2e-dev -- cart-checkout-translations`
3. Check that the tests that run after this one pass and language is switched back to English successfully.

### CODE REVIEW

File: tests/e2e/specs/shopper/cart-checkout-translations.test.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+	it( 'should be able to view translated Cart block ', async () => {
+		// [x] Install woo-blocks plugin using CLI
+		// [x] Install language for all plugins
+		// [x] Navigate to the Settings page
+		// [x] Change language to German
+		// [x] Navigate to Shop page
+		// [x] Add product to cart
+		// [x] Go to CArt page and check all translated fields
+		// [] Go toCheckout and check all translated fields
```
Comment (nielslange) [inline tests/e2e/specs/shopper/cart-checkout-translations.test.js]: Guess we can remove these comments, can't we? I also noticed that the comment says `Change language to German`, but the test itself is using French.

File: tests/utils/run-cli-from-test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+		exec( `${ cmd } ${ args.join( ' ' ) }`, ( error, stdout, stderr ) => {
+			resolve( {
+				code: error && error.code ? error.code : 0,
+				error,
+				stdout,
+				stderr,
+			} );
+		} );
+	} );
+}
```
Comment (nielslange) [inline tests/utils/run-cli-from-test.js:17]: Thank you so much for implementing this! This is extreme useful! 👏

**Review by nielslange — APPROVED** [review]
LGTM! I only left a comment regarding removing a few comments. That said, the PR is good to be merged and shipped. 🙌

File: tests/e2e/specs/shopper/cart-checkout-translations.test.js (hunk 1 above)
Comment (alexflorisca) [reply tests/e2e/specs/shopper/cart-checkout-translations.test.js]: Oh yeah, good spot, thanks!

**Comment by tarhi-saad** [comment]
@alexflorisca, we've merged this PR to the trunk branch with some falling E2E tests in CI. See [E2E tests results of this PR](https://github.com/woocommerce/woocommerce-gutenberg-products-block/runs/5722820696?check_suite_focus=true):

```
Summary of all failing tests
FAIL tests/e2e/specs/shopper/cart-checkout-translations.test.js (23.349 s)
  ● Shopper → Cart → Can view translated cart & checkout blocks › should be able to view translated Cart block 

    TimeoutError: Text not found "Produit"

      waiting for function failed: timeout 500ms exceeded

  ● Shopper → Cart → Can view translated cart & checkout blocks › should be able to view translated Checkout block

    TimeoutError: Text not found "Coordonnées"

      waiting for function failed: timeout 500ms exceeded


Test Suites: 1 failed, 39 passed, 40 total
Tests:       2 failed, [136](https://github.com/woocommerce/woocommerce-gutenberg-products-block/runs/5722820696?check_suite_focus=true#step:10:136) passed, [138](https://github.com/woocommerce/woocommerce-gutenberg-products-block/runs/5722820696?check_suite_focus=true#step:10:138) total
```

**Comment by alexflorisca** [comment]
Ok thanks for pointing it out @tarhi-saad, I'll take a look!

**Comment by alexflorisca** [comment]
Fixed in #6154

**Pull request merged**